### PR TITLE
[ZH] Translation of 11-2

### PR DESCRIPTION
**Pull request by watchernyu**
ZH translation of 11-2 is now finished. All original English content are commented out block by block.

### CODE REVIEW

File: docs/zh/week11/11-2.md [hunk 1]
```diff
@@ -0,0 +1,526 @@
+---
+lang-ref: ch.11-2
+lecturer: Yann LeCun
+title: Loss Functions (cont.) and Loss Functions for Energy Based Models
```
Comment (Atcold) [inline docs/zh/week11/11-2.md:4]: Title needs translation.
`lang: zh` is missing.

File: docs/zh/week11/11-2.md [hunk 2]
```diff
@@ -0,0 +1,526 @@
...
+authors: Charles Brillo-Sonnino, Shizhan Gong, Natalie Frank, Yunan Hu
+date: 13 April 2020
```
Comment (Atcold) [inline docs/zh/week11/11-2.md:6]: Use `Apr` instead of `April`.
`translator:` and `translation-date:` are missing.

**Review by Atcold — CHANGES_REQUESTED** [review]
You need to update `docs/_config.yml` and fix what I've pointed out in the comments.
Thanks for your contribution!

File: docs/zh/week11/11-2.md (hunk 1 above)
Comment (JonathanSum) [reply docs/zh/week11/11-2.md:4]: title: 損失函數(連續性) 和用在能量基礎模型上的損失函數

I am assuming the cont. stands for continuous.

Comment (watchernyu) [reply docs/zh/week11/11-2.md:4]: Thanks for the comments! Modifying them now.

File: docs/zh/week11/11-2.md (hunk 2 above)
Comment (watchernyu) [reply docs/zh/week11/11-2.md:6]: Thanks! Just added!

**Review by Atcold — APPROVED** [review]
Thanks! Edited `docs/_config.yml` for you.

**Pull request merged**